docs/introduction.rst

```diff
@@ -53,127 +56,143 @@ Survey of North America (BBS) and the Alwyn Gentry Tree Transect data:
+        edit_json           CLI to edit retriever datapackage.json script
+        delete_json         CLI to remove retriever datapackage.json script
+        ls                  display a list all available dataset scripts
+        citation            view citation
+        reset               reset retriever: removes configation settings,
+                            scripts, and cached data
+        help
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -v, --version         show program's version number and exit
+      -q, --quiet           suppress command-line output
+
+To install datasets, use the ``install`` command.
+
+Examples
+~~~~~~~~
+
+**Using install**
+
+$ ``retriever install -h`` (gives install options)
+
+::
 
     usage: retriever install [-h] [--compile] [--debug]
-                             {mysql,postgres,sqlite,msaccess,csv} ...
+                             {mysql,postgres,sqlite,msaccess,csv,json,xml} ...
 
     positional arguments:
-      {mysql,postgres,sqlite,msaccess,csv}
+      {mysql,postgres,sqlite,msaccess,csv,json,xml}
                             engine-specific help
         mysql               MySQL
         postgres            PostgreSQL
         sqlite              SQLite
         msaccess            Microsoft Access
         csv                 CSV
+        json                JSON
+        xml                 XML
 
     optional arguments:
       -h, --help            show this help message and exit
       --compile             force re-compile of script before downloading
       --debug               run in debug mode
 
 
-Examples
-~~~~~~~~
+**Examples using install with engines**
 
 
+These examples are using Breeding Bird Survey data (Breed-bird-survey)
+Using a specific database engine. The retriever has support for various engines; mysql, postgres, sqlite, msaccess, csv, json, xml, download_only
 
-These examples are using Breeding Bird Survey data (BBS)
+Install data into Mysql::
 
-Using Install::
+   retriever install mysql –user myusername –password ***** –host localhost –port 8888 –database_name testdbase Breed-bird-survey
+   retriever install mysql -user (using attributes in the client authentication configuration file)
 
-   retriever install -h (gives install options)
+Install data into postgres::
 
-Using a specific database engine. The retriever has support for various engines; mysql, postgres, sqlite, msaccess, csv, download_only::
+   retriever install postgres –user myusername –password ***** –host localhost –port 5432 –database_name testdbase Breed-bird-survey
+   retriever install postgres Breed-bird-survey (using attributes in the client authentication configuration file)
 
-   retriever install {Engine}
+Install data into sqlite::
 
-   retriever install mysql -h     ..(gives install mysql options)::
+   retriever install sqlite Breed-bird-survey -f mydatabase.db (will use or create default mydatabase.db)
```

Comment:
I don't understand what `default` is communicating here. I'd probably remove it, but let me know if I'm missing something.